packages/inventory/src/Stocktake/api/hooks.ts

```diff
@@ -182,9 +182,9 @@ export const useStocktakeItems = (): UseQueryResult<StocktakeSummaryItem[]> => {
       ([itemId, lines]) => {
         return {
           id: itemId,
-          item: lines[0].item,
+          item: lines[0]?.item,
```

Comment:
I had to remove the `[T, ...T[]]` type syntax and as a result, cope with potentially undefined array elements.
Once again, not sure why ts is unhappy now, but couldn't find an alternative workaround